javascript/javascript1/week2/jsweek2.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lord " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2 = getFullname("Benjamin", "Hughes", false);
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+
+// Event application
+const wDays7 = [
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+];
+function getEventWeekday(daysOfEvent) {
+    const fullD = new Date();
+    const manyDaysFromToday = new Date(
+        fullD.getDate(),
+        fullD.getMonth() + daysOfEvent
+    );
+    return wDays7[manyDaysFromToday.getDay()];
+}
+// With todays weekday a tuesday
+console.log(getEventWeekday(9)); // Logs out "Thursday"
+
+// With todays weekday a Friday
+console.log(getEventWeekday(5)); // Logs out "Sunday"
+
+
+
+// Weather wear
+function weatherWear(temperature) {
+    if (temperature < 30) {
+        temperature = "Jaket in cold weather";
+    } else {
+        temperature = "T-shirt in hot weather";
+    }
+    return `Hi you should wearing a ${temperature}`;
+}
+console.log(weatherWear(10));
```

Comment:
Good work. I like that its either hot or cold in your solution 😊

It would be good to focus on naming conventions here. Right now you are using temperature as a parameter for your function, and you apply this value in the condition for your if else statement - really good, that makes sense. ✨

However, I would suggest you declare a local variable for the function, e.g. 'youShouldWearThis' instead of using temperature for the string variable. I have exemplified it below; 

function weatherWear(temperature) {
let youShouldWearThis ;
if (temperature < 30) {
youShouldWearThis = "Jaket in cold weather"
}

Does that make sense? 😊
Great to see you experiment with template literals 👌